Our Scripts/EWvsZ.py

```diff
@@ -24,4 +25,9 @@
 
 EW=[mod.EW(i) for i in z]
 
-plotter.plot(z,EW,"A plot of the equivalent width against redshift","Redshift","EW [Å]",path)
\ No newline at end of file
+x = pylab.array([2.5,2.8,2.9,3.1, 3.3, 3.7, 4.1, 4.5, 4.8, 5.0, 5.3])
+y = pylab.array([117.134349876, 122.113769531, 172.679885864, 143.5936203, 149.371124268, 96.3648490905, 189.002449036, 181.127731323, 95.0448436864, 125.935153962, 143.343048096])
+
+plt.scatter(x, y)
+plt.plot(z, EW )
+plt.show()
```

Comment:
Is this section meant to replace the master?

Reply:
yeah, it means we can plot data points on top of the graph